packages/driver/types/spec-types.d.ts

```diff
@@ -0,0 +1,8 @@
+// NOTE: This is for internal Cypress spec types that exist in support/utils.js for testing convenience and do not ship with Cypress
+
+declare namespace Cypress {
+  interface Chainable {
+    getAll(...aliases: string[]): Chainable
+    shouldWithTimeout(cb: (subj: {}) => void, timeout?: number): Chainable
+  }
+}
```

Comment:
```suggestion
}

```

Reply:
updated in [c264f20](https://github.com/cypress-io/cypress/pull/20949/commits/c264f206d6ffcdfec536f4e2800f20b9a59bfcf6)